packages/govuk-frontend-review/.browserslistrc

```diff
@@ -1,8 +1,23 @@
-> 0.1%
-last 2 Chrome versions
-last 2 Firefox versions
-last 2 Edge versions
+[javascripts]
+> 0.1% in GB and supports es6-module
```

Comment:
Is there any way to avoid needing to duplicate this here and in the webpack example?

Reply:
@romaricpascal has asked me to use separate configs in the past. Another one for a future npm package?